sources/parsley/main.py

```diff
@@ -121,10 +121,7 @@ def main():
     else:
         parser = parsley.parse_usb_debug
 
-    if args.format == "telemetry":
-        channel = "telemetry/" + RECEIVE_CHANNEL
-    else:
-        channel = RECEIVE_CHANNEL
+    channel = RECEIVE_CHANNEL
```

Comment:
Pls just delete the channel variable and update where it's used directly

Reply:
Done. Functionally though, is it correct?